### ISSUE TITLE
Your instructions are all wrong.

### ISSUE THREAD

**Issue by rica01** [derekjanni/PyOCR2#1]
The installation fails, your program has a looooooooooooooooot more dependencies than it shows here. Its missing files (datalabels.csv), etc.

It showed a lot of promise but if no one can use this software, its worth nothing. Try to fix it! :+1:

**Comment by derekjanni** [comment]
You're right, I haven't give the documentation the care it deserves.. I'll give it some TLC and see if I can fix it...
